jsx_test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Card from "./card";
+
+export default class CardList extends React.Component {
+  constructor(props)
+  {
+    super(props);
+  }
+  render () {
+    var card_list = this.props.deck_list.map(function(card)
```

Comment:
Unexpected var, use let or const instead.<br>Identifier 'card_list' is not in camel case.<br>'deck_list' is missing in props validation<br>'deck_list.map' is missing in props validation<br>Opening curly brace does not appear on the same line as controlling statement.<br>Missing function expression name.<br>Missing space before function parentheses.